sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp

```diff
@@ -954,7 +954,8 @@ namespace Azure { namespace Storage { namespace Blobs {
       Azure::Core::DateTime CreatedOn;
       Azure::Core::Nullable<Azure::Core::DateTime> ExpiriesOn;
       Azure::Core::Nullable<Azure::Core::DateTime> LastAccessedOn;
-      Azure::Core::Nullable<std::string> ContentRange;
+      Azure::Core::Http::Range ContentRange;
+      int64_t BlobSize = 0;
```

Comment:
Should this be nullable?

Reply:
Under what circumstances it's null?

Reply:
I think some parts can be null in the HTTP standard. However, since you mentioned that the Blob service is never going to return a null part in this string, I'm OK. Let's deal with it if it occurs in the future.